docker-bake.hcl

```diff
@@ -0,0 +1,39 @@
+variable "GO_VERSION" {
+  default = "1.18"
+}
+
+target "_common" {
+  args = {
+    GO_VERSION = GO_VERSION
+    BUILDKIT_CONTEXT_KEEP_GIT_DIR = 1
+  }
+}
+
+// Special target: https://github.com/docker/metadata-action#bake-definition
+target "docker-metadata-action" {}
+
+
+group "default" {
+  targets = ["image-local"]
+}
+
+target "image" {
+  inherits = ["_common", "docker-metadata-action"]
+}
+
+target "image-local" {
+  inherits = ["image"]
+  output = ["type=docker"]
+}
+
+target "image-all" {
+  inherits = ["image"]
+  platforms = [
+    "linux/amd64",
+    "linux/arm/v6",
```

Comment:
BuildKit doesn't provide the release image for this platform so we cannot provide it.

Reply:
Thanks for the information 🫶